packages/cli/package.json

```diff
@@ -18,7 +18,7 @@
   "type": "commonjs",
   "main": "src/index.ts",
   "bin": {
-    "coauthors-cli": "./dist/index.js"
+    "@coauthors/cli": "./dist/index.js"
```

Comment:
@2-NOW Check this bin please. is it okay for @coauthros/cli?

Reply:
Hmm, I'm not sure if this is a good, easy to use cli command. I also feel like `@coauthors/cli` is hard to use as a command, what do you think?

Reply:
If we should choose coauthors-cli vs @coauthors/cli, In my opinion, I will choose @coauthors/cli.
I wanted to make library users don't care what command is to use @coauthors/cli. so I thought unifying all thing to use @coauthors/cli is better than use coauthors-cli.

Anyway I will revert this change. because this is not point of this Pull Request. Thanks

Reply:
I reverted it now, and I lefted issue from this comment